### PR TITLE
NEW: adds several guides, tidies documentation

### PR DESCRIPTION
**Pull request by glennr**

### CODE REVIEW

**Comment by glennr** [comment]
[Getting Started: Errors (0.1)](https://trello.com/c/l6mcq0Q8/48-getting-started-errors-01)

**Comment by glennr** [comment]
[document search syntax](https://trello.com/c/u2fAxkBz/139-document-search-syntax)

File: guides/global-node-ids.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          },
+        ]
+      }
+    }
+  }
+```
+
+## Fetching a node by ID
+
+In almost all cases, we provide a query for individual nodes.
```
Comment (marcinbunsch) [inline guides/global-node-ids.md:58]: What are the exceptions?

**Review by marcinbunsch — APPROVED** [review]
This is good, left one question on exceptions to fetching nodes by id

File: guides/global-node-ids.md (hunk 1 above)
Comment (glennr) [reply guides/global-node-ids.md:58]: I was thinking about Messages when I wrote that e.g. (`<changeset-id>-full`).

**Comment by glennr** [comment]
merged manually

**Pull request closed without merging**